vizro-core/tests/unit/vizro/conftest.py

```diff
@@ -43,6 +42,4 @@ def dashboard(page1, page2):
 
 @pytest.fixture()
 def dashboard_prebuild(dashboard):
```

Comment:
Is this fixture even used now? I think it was replaced by the Vizro.pre_build?

Reply:
It is used in the `dashboard` and `navigation` tests

Reply:
Ah yeah, I see. I think in the dashboard it might be better to actually test the function so change to:

In the dashboard tests, shall we actually explicitly call the `dashboard.pre_build` given that it's testing that? Feel free to ignore if you think it doesn't matter much

```
@pytest.mark.usefixtures("vizro_app", "dashboard")
    def test_dashboard_page_registry(self, mock_page_registry, dashboard):
        dashboard.pre_build()
        result = dash.page_registry
        expected = mock_page_registry
```

Reply:
Yes I think that would be cleaner!